MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -998,6 +1000,36 @@ - (RoomThreadedReceiptsStore*)getOrCreateRoomThreadedReceiptsStore:(NSString*)ro
     return threadedStore;
 }
 
+- (void)setUnreadMarkerForRoom:(nonnull NSString*)roomId;
```

Comment:
If these 2 methods just call `super` there is no need to override them, and can be safely deleted.